templates/default/sentinel.conf.erb

```diff
@@ -15,6 +15,8 @@ syslog-facility <%= @syslogfacility %>
 # The port that this sentinel instance will run on
 port <%=@sentinel_port%>
 
+<% @masters.each do |current| %>
```

Comment:
What do you think about including the comments in each iteration?  Would this make the property file harder to read? Realistically does the readability even matter?